src/features/bookOffer/services/useReviewInAppInformation.ts

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react'
+
+import { storage } from 'libs/storage'
+
+export const useReviewInAppInformation = () => {
+  const [timesReviewHasBeenRequested, setTimesReviewHasBeenRequested] = useState(0)
+  const [shouldReviewBeRequested, setShouldReviewBeRequested] = useState(false)
+  const currentDate = Date.now()
+  const oneYearInMilliseconds = 31536000000
```

Comment:
```suggestion
  const ONE_YEAR = 365 * 24 * 60 * 60 * 1000
```

et tu peux le sortir du hook?